### ISSUE TITLE
Bootstrap tooltip disappears immediately

### ISSUE THREAD

**Issue by eappell** [zeroclipboard/zeroclipboard#369]
I'm trying to apply a tooltip to elements that trigger a copy using ZC, but when I mouse over those elements, the tooltip appears for just a fraction of a second and then fades away.  I'm not sure how to get this to work, but if anyone has a solution for this, please share the code.  I've tried using the css effects classes, but when I apply the tooltip to them I get no tooltip at all...

Thanks!

**Comment by ttseng** [comment]
Did this ever get resolved?  I'm running into the same issue.

**Comment by JamesMGreene** [comment]
Nope, watch #463 for more updates when I eventually find time again. :sad:

**Comment by Tigerman55** [comment]
Is this solved yet? If so, do you have a link to a webpage which the answer? I've looked everywhere for this.

**Comment by JamesMGreene** [comment]
@Tigerman55: Really? You looked everywhere? Like... at the comment immediately above yours that was ~27 hours before your comment? :stuck_out_tongue_winking_eye: 

Long-story-short: no, not resolved, and I can't find any free time to work on ZeroClipboard (or jQuery, or PhantomJS, etc.) lately. Believe me, that makes _me_ very sad indeed. :cry:

**Comment by Tigerman55** [comment]
I found a few issues with my own code. First, I wasn't always including the bootstrap.js file when I was trying to use tooltips. Second, it seemed to work better when I didn't have the console open.

**Comment by Pashted** [comment]
i came up with a simple solution. not perfect, but it works.

<b>step 1</b> is editing bootstrap.js (or bootstrap.min.js) - remove this stroke from script... 
<code>this.options.animation&&t.addClass("fade"),</code>

or use attribute <code>data-animation="false"</code> in link with tooltip.

<b>step 2</b> (not necessary, because after 1st step "blinking" has gone)
edit bootstrap css file by adding following 
<code>.tooltip {
    transition: opacity .1s ease-out;
}</code>

**Comment by gritfish** [comment]
I know this is an incredibly old report but I found the tooltip disappearing issue was resolved immediately by adding "pointer-events: none;" to the tooltip class in the css.

**Comment by yitznewton** [comment]
> I know this is an incredibly old report but I found the tooltip disappearing issue was resolved immediately by adding "pointer-events: none;" to the tooltip class in the css.

@gritfish you just won our day here at our shop :sparkling_heart:
